stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/MessagesActivity.kt

```diff
@@ -78,7 +78,6 @@ class MessagesActivity : AppCompatActivity() {
             ChatTheme(dateFormatter = ChatApp.dateFormatter) {
                 MessagesScreen(
                     channelId = channelId,
-                    messageLimit = 30,
```

Comment:
Not necessary as the default argument for `messageLimit` in `MessagesScreen` is the same